eng/pipelines/libraries/build-job.yml

```diff
@@ -49,23 +51,21 @@ jobs:
       name: build
       displayName: 'Build'
 
-      ${{ if ne(parameters.liveRuntimeBuildConfig, '') }}:
-        dependsOn:
-        # Use full product dependency for non-coreclr and test builds
-        - ${{ if or(ne(parameters.runtimeFlavor, 'coreclr'), ne(parameters.testScope, '')) }}:
-          - ${{ format('{0}_{1}_product_build_{2}{3}_{4}_{5}', parameters.runtimeFlavor, parameters.runtimeVariant, parameters.osGroup, parameters.osSubgroup, parameters.archType, parameters.liveRuntimeBuildConfig) }}
-
       variables:
+        - librariesTestsArtifactName: ${{ format('libraries_test_assets_{0}_{1}_{2}', parameters.osGroup, parameters.archType, parameters.buildConfig) }}
         - _subset: libs
         - _additionalBuildArguments: ''
         - ${{ parameters.variables }}
         - ${{ if eq(parameters.osGroup, 'Browser') }}:
           - EMSDK_PATH: /usr/local/emscripten
-        # for coreclr library builds (when not testing) build corelib as well.   
-        - ${{ if and(eq(parameters.runtimeFlavor, 'coreclr'), eq(parameters.testScope, '')) }}:
+        - ${{ if eq(parameters.runtimeFlavor, 'coreclr') }}:
           - _subset: clr.corelib+libs
+        - ${{ if eq(parameters.runtimeFlavor, 'mono') }}:
+          - _subset: mono.corelib+libs
```

Comment:
We don't need to build corelib anymore manually as we now have a P2P to it.

Reply:
That's true. I'll remove that subset 😀

Reply:
Fixed, thanks.